index.html

```diff
@@ -4,11 +4,13 @@
         <!-- include jsPsych first -->
         <script src="https://web-experiments.lab.hum.uu.nl/jspsych/6.1.0/jspsych.js"></script>
         <script src="https://web-experiments.lab.hum.uu.nl/jspsych/6.1.0/plugins/jspsych-html-button-response.js"></script>
-        <script src="jspsych-uil-utils.js" type="text/javascript"></script>
+        <script src="./jspsych-uil-utils-import.js" type="module"></script>
     </head>
     <body>
         <p>For now you can play with the browser console to examine the jspsych uil utils</p>
-    <script>
+    <script type="module">
```

Comment:
This can be reverted right?

Reply:
Yes I think this may be a regular script tag as all modular stuff happens in jspsych-uil-utils-import.js. This is probably a result of me playing with the utils in order to make the utils themself modular. I can revert it :-)

Reply:
8b41b48